runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/WoodcuttingConfig.java

```diff
@@ -99,4 +99,70 @@ default boolean showRespawnTimers()
 	{
 		return true;
 	}
+
+	@ConfigItem(
```

Comment:
I think adding a config section for forestry would be nice to help keep it separate from general woodcutting settings.